test/integration/base_integration_test.rb

```diff
@@ -0,0 +1,13 @@
+require 'test_helper'
+require 'mocha/setup'
+
+##
+# The base class for all integration tests. Which includes operations common to all integration tests.
```

Comment:
Not trying to be super nitpicky, but technically "Which includes operations comment to all integration tests." isn't a sentence. All you have to do here is remove the period after "tests" and de-capitalize the "W" of "Which".

Reply:
This should now be resolved.